modAionImpl/src/org/aion/zero/impl/cli/Cli.java

```diff
@@ -481,6 +582,10 @@ private String readPasswordFromReader(String prompt, BufferedReader reader) {
         return RecoveryUtils.revertTo(block);
     }
 
+    public static boolean isValid(String value) {
+        return !value.isEmpty() && !value.matches(".*[-=+,.?;:'!@#$%^&*].*");
```

Comment:
Could you maybe include a short comment on what this regex is matching and why -- for the next dev to come along.

Reply:
sure